itou/www/job_seekers_views/views.py

```diff
@@ -220,27 +220,47 @@ def get(self, request, *args, **kwargs):
         return HttpResponseRedirect(reverse(view_name, kwargs={"session_uuid": self.job_seeker_session.name}))
 
 
-class JobSeekerBaseView(TemplateView):
+class ExpectedJobSeekerKindMixin(TemplateView):
```

Comment:
```suggestion
class ExpectedJobSeekerSessionMixin(TemplateView):
```
et on pourra même éventuellement enlever le `JobSeeker` dans le futur en renommant `self.job_seeker_session` en `self.session_namespace`

Reply:
Oui :+1: 
Un pas de plus vers une généralisation ?